src/geometry/manifold/instances/sphere.lean

```diff
@@ -0,0 +1,267 @@
+/-
+Copyright (c) 2021 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import geometry.manifold.charted_space
+import analysis.normed_space.inner_product
+
+/-!
+# Manifold structure on the sphere
+
+This file defines stereographic projection from the sphere in an inner product space `E`, and uses
+it to put a charted space structure on the sphere.
```

Comment:
Could you expand a little bit the docstring, by giving the name of the stereographic projection, saying that this is a local homeomorphism between the complement of a point and the orthogonal hyperplane, and that the charted space structure is obtained by postcomposing with isometries. And mention the `fact` mechanism in an implementation section?